ipatests/test_integration/test_caless.py

```diff
@@ -1181,6 +1188,23 @@ def test_no_ds_password(self):
         if self.domain_level > DOMAIN_LEVEL_0:
             self.verify_installation()
 
+    @pytest.mark.xfail(reason='Ticket N 7274')
+    @replica_install_teardown
+    def test_certs_with_no_password(self):
+        # related to https://pagure.io/freeipa/issue/7274
+
+        self.create_pkcs12('ca1/server', filename='http.p12', password='',
+                           no_pin=True)
+        self.create_pkcs12('ca1/server', filename='dirsrv.p12', password='',
+                           no_pin=True)
+        self.prepare_cacert('ca1')
+
+        self.prepare_replica(http_pkcs12='http.p12',
+                             dirsrv_pkcs12='dirsrv.p12',
+                             http_pin=None, dirsrv_pin=None)
```

Comment:
In non-interactive mode, I believe you need to provide --http-pin="" and --dirsrv-pin="".